bundle/Templating/Twig/Extension/SiteRuntime.php

```diff
@@ -117,4 +119,12 @@ public function getImageUrl(Content $content, string $fieldIdentifier, string $a
 
         return '/';
     }
+
+    public function calculateReadingTime(string $text): float
+    {
+        $wordCount = str_word_count($text);
+        $readingTime = floor($wordCount / self::WORDS_PER_MINUTE);
+
+        return $readingTime === false || $readingTime < 1 ? 1 : $readingTime;
```

Comment:
Can we cast to and return an integer? `floor` function says: `Returns the next lowest integer value (as float)`. So if it returns an integer, why don't we make it explicit here that it is indeed an integer?

Reply:
Changed in 99eff3b